Flagsmith.FlagsmithClient/FlagsmithClient.cs

```diff
@@ -358,7 +358,7 @@ private async Task<IFlags> GetIdentityFlagsFromApi(string identity, List<ITrait>
 
                 if (traits != null && traits.Count > 0)
                 {
-                    jsonBody = JsonConvert.SerializeObject(new { identifier = identity, traits });
+                    jsonBody = JsonConvert.SerializeObject(new { identifier = identity, traits, transient });
                     jsonResponse = await GetJson(HttpMethod.Post, url, body: jsonBody).ConfigureAwait(false);
                 }
                 else
```

Comment:
We need to add support for transient traits here and cover it with tests as well
```
                else
                {
                    url += $"?identifier={identity}" + transient ? "&transient=false" : "";
                    jsonResponse = await GetJson(HttpMethod.Get, url).ConfigureAwait(false);
                }
```